spring-kafka/src/main/java/org/springframework/kafka/core/KafkaAdmin.java

```diff
@@ -213,6 +213,14 @@ public void setClusterId(String clusterId) {
 		this.clusterId = clusterId;
 	}
 
+	/**
+	 * Get the clusterId property.
+	 * @since 3.3.0
```

Comment:
Since we have agreed that this is a bug and there is no easy workaround (but fix is simple), then this one is going to be back-ported to supported versions. Therefore this `@since` is `3.1.8`